src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -102,7 +103,9 @@ DeviceTypes devicesType = {
     std::make_pair(std::string("CARPLAY_WIRELESS_IOS"),
                    hmi_apis::Common_TransportType::WIFI),
     std::make_pair(std::string("CLOUD_WEBSOCKET"),
-                   hmi_apis::Common_TransportType::CLOUD_WEBSOCKET)};
+                   hmi_apis::Common_TransportType::CLOUD_WEBSOCKET),
+    std::make_pair(std::string("WEBENGINE_WEBSOCKET"),
```

Comment:
There is a mix of terminology for things named `webengine` vs `local`. Can we choose one to use consistently? I think i prefer `webengine` instead of `local`.

Reply:
@JackLivio It is not a mix. WebEngine is a device, as transport is relatedto the device(not to the applications) - it is WEBENGINE WEBSOCKET. And all applications from WebEngine device considered to be local. The same is stated in the original proposal.

Reply:
Thank you for the explanation.